api/v1/serializers/players-serializer.js

```diff
@@ -24,12 +24,23 @@ _.forEach(playerResourceKeys, (key, index) => {
 });
 playerResourceKeys.push('playerCards');
 
-const playerConverter = (rawPlayers) => {
-  _.forEach(rawPlayers, (player) => {
-    player.MEMBER_LEVEL = parseInt(player.MEMBER_LEVEL, 10);
-    player.MEMBER_EXP_OVER_LEVEL = parseInt(player.MEMBER_EXP_OVER_LEVEL, 10);
-    player.PLAYER_BET = parseInt(player.PLAYER_BET, 10);
+const playerConverter = (player) => {
+  player.MEMBER_LEVEL = parseInt(player.MEMBER_LEVEL, 10);
+  player.MEMBER_EXP_OVER_LEVEL = parseInt(player.MEMBER_EXP_OVER_LEVEL, 10);
+  player.PLAYER_BET = parseInt(player.PLAYER_BET, 10);
+};
+
+const mergeRawPlayers = (rawPlayers) => {
+  const groupedRawPlayers = _.groupBy(rawPlayers, 'PLAYER_ID');
+  const mergedRawPlayers = _.map(groupedRawPlayers, (playerMetaDataArray) => {
+    playerMetaDataArray[0].playerCards = playerMetaDataArray[0].CARD_NUMBER === null ? []
+      : _.map(playerMetaDataArray, data => ({
+        cardNumber: data.CARD_NUMBER,
+        cardSuit: data.SUIT,
+      }));
+    return playerMetaDataArray[0];
   });
+  return mergedRawPlayers;
 };
 
 const serializePlayers = (rawPlayers, query, gameId) => {
```

Comment:
Shouldn't this function also be calling `mergeRawPlayers`?

Reply:
It's mergedRawPlayers.